contribution/Killercoda-Kubearmor/kubearmor-installation-view.sh

```diff
@@ -0,0 +1,17 @@
+cat > setup.sh << 'EOL'
+#!/bin/bash
+namespace="kubearmor"
+duration=120
+watch -n 1 "kubectl get po -n $namespace" &
+watch_pid=$!
+sleep $duration
+while kubectl get po -n $namespace | tail -n +2 | awk '{print $3}' | grep -q -v "Running"; do
+  sleep 8
+done
```

Comment:
```suggestion
kubectl wait --for=condition=ready --timeout=$duration -n kubearmor pod -l kubearmor-app
```

Reply:
Thanks for the suggestion. changes done.